app/src/main/java/com/vitaliidiadchenko/mykotkinapplication/FragmentMovieDetail.kt

```diff
@@ -53,18 +61,47 @@ class FragmentMovieDetail : Fragment() {
         listener = null
     }
 
-    private fun updateData() {
-        (recyclerView?.adapter as? ActorViewHolderAdapter)?.apply {
-            bindActors(listActors)
+    private fun setupView(movie: Movie) {
+        view?.run {
```

Comment:
Идеоматически, правильнее в данном случае использовать `let` https://kotlinlang.org/docs/idioms.html#execute-if-not-null
```suggestion
        view?.let {
```

Reply:
Не открывается ссылка..
Путаюсь пока еще в этих функциях run, with, apply ...ect.
Поставил let, метод findViewById, выделился красным.
Почитал статью, написали что при инициализации view используйте apply..
http://developer.alexanderklimov.ru/android/kotlin/with-apply-also.php
Не знаю конечно можно ли верить этому источнику..

Reply:
Хороший вариант. Источнику можно доверять)